explainaboard/loaders/loader_factory.py

```diff
@@ -31,6 +31,7 @@
 from explainaboard.loaders.tabular_regression import TabularRegressionLoader
 from explainaboard.loaders.text_classification import TextClassificationLoader
 from explainaboard.loaders.text_pair_classification import TextPairClassificationLoader
+from explainaboard.loaders.text_to_sql import TextToSQLLoader
```

Comment:
I think this always requires additional installation. I think we could no longer separate this loader from others, and the following installation flag needs to be integrated into the default.

https://github.com/neulab/ExplainaBoard/blob/0a9b3bcefefcb96751fa3d663d24fff2df9dd250/setup.cfg#L39

Reply:
Thank you for the review! Do you want me to create a separate PR to merge the required installations into default?

Reply:
No, it would be reasonable to mix it into this request I think.

Reply:
Mixed it here. Thank you!